contracts/src/v0.8/dev/special/NoCancelVRFCoordinatorV2.sol

```diff
@@ -0,0 +1,883 @@
+// SPDX-License-Identifier: MIT
```

Comment:
I wish we could somehow run the hardhat tests on both these files...

Reply:
That's not a bad idea, we'll just have to exclude the tests that call `cancelSubscription` and `ownerCancelSubscription`.